charts/merlin/values.yaml

```diff
@@ -236,6 +236,17 @@ config:
       Acks: 0
       LingerMS: 100
       AdditionalConfig: "{}"
+  ObservabilityPublisher:
+    KafkaConsumer:
+      Brokers: "kafka-brokers"
+    DefaultResources:
+      Requests:
+        CPU: "1"
+        Memory: "1Gi"
+      Limits:
+        CPU: "2"
+        Memory: "1Gi"
+    EnvironmentName: id-dev
```

Comment:
I had to add these dummy configs in because there were validation checks on these fields:
- https://github.com/caraml-dev/helm-charts/actions/runs/11888073545/job/33129431665#step:12:336
- https://github.com/caraml-dev/helm-charts/actions/runs/11904740287/job/33174055464#step:12:334